sources/Debate-frontend/src/app/app.component.html

```diff
@@ -17,11 +17,31 @@
         notifications
       </mat-icon>
     </button>
-    <button mat-button class="login-button" routerLink="/login" >
-      <mat-icon class="toolbar-icon login-user">
-          account_circle
-      </mat-icon>
-    </button>
+    <div *ngIf="isLogged; then thenBlock else elseBlock"></div>
```

Comment:
again, thenBlock could be more meaningful, i.e loggedSection, notLoggedSection (or something better)